apps/server/src/modules/common-cartridge/service/common-cartridge-export.service.ts

```diff
@@ -31,4 +34,10 @@ export class CommonCartridgeExportService {
 
 		return courseRooms;
 	}
+
+	public async findAllCardsByIds(ids: Array<string>): Promise<CardListResponseDto> {

```

Comment:
this method is not covered by a test, please implement a test for it

Reply:
Done